css/aboutindex.css

```diff
@@ -0,0 +1,231 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
```

Comment:
You can also put the reset/normalize css code in a separate file so you don't have to copy and paste it into different CSS files.